### ISSUE TITLE
ensure: "All binaries are checked with antivirus before publishing."

### ISSUE THREAD

**Issue by jakoch** [WPN-XM/WPN-XM#295]
**Currently, i do AV checks with Avira and Kaspersky.**

To make this AV check even more transparent, we want this (a) as a separate build step
and (b) with published scan reports on the website.

- add build step
- do a virus check on the whole releases folder, before the files are published
- maybe automatical virus check and reports for all releases via the VirusTotalAPI
    - https://github.com/jayzeng/virustotal_apiwrapper
- if everything is ok (or only known bogus messages / false positives), then release files
- add the virus reports next to the downloads

In other words, ensure: 
- [ ] "All binaries are checked with antivirus before publishing."
- [ ] "All reports are available."
- [ ] Add Message: "Virus Status: Clean (Clean) by ClamAV & VirusTotal. Note: Use virus status as a reference, rescan with your own Anti-Virus."

**Comment by jakoch** [comment]
Will not implement, because the stack is no longer build on a local machine, but build transparently on Travis-Ci and deployed to Github. 

Contributions with a focus on additional AV checks are welcome.
Closed for now.
